### DISCUSSION TITLE
Release 2.0.0 Beta

### DISCUSSION THREAD

**Discussion by IacobIonut01** [IacobIonut01/Gallery#158] [Announcements]
## What's Changed
* Update hilt to v2.47 by @renovate in https://github.com/IacobIonut01/Gallery/pull/108
* Update plugin baselineProfilePlugin to v1.2.0-beta02 by @renovate in https://github.com/IacobIonut01/Gallery/pull/111
* Update kotlinCoroutinesVersion to v1.7.3 by @renovate in https://github.com/IacobIonut01/Gallery/pull/115
* Update dependency androidx.benchmark:benchmark-macro-junit4 to v1.2.0-beta02 by @renovate in https://github.com/IacobIonut01/Gallery/pull/110
* Update dependency gradle to v8.2.1 by @renovate in https://github.com/IacobIonut01/Gallery/pull/105
* Update glideVersion to v4.16.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/147
* Update plugin baselineProfilePlugin to v1.2.0-beta04 by @renovate in https://github.com/IacobIonut01/Gallery/pull/129
* Update dependency gradle to v8.3 by @renovate in https://github.com/IacobIonut01/Gallery/pull/146
* Update dependency androidx.benchmark:benchmark-macro-junit4 to v1.2.0-beta04 by @renovate in https://github.com/IacobIonut01/Gallery/pull/127
* New Crowdin updates by @IacobIonut01 in https://github.com/IacobIonut01/Gallery/pull/120
* Update actions/checkout action to v4 by @renovate in https://github.com/IacobIonut01/Gallery/pull/153
* Update dependency androidx.benchmark:benchmark-macro-junit4 to v1.2.0-beta05 by @renovate in https://github.com/IacobIonut01/Gallery/pull/150
* Update agp to v8.2.0-beta04 by @renovate in https://github.com/IacobIonut01/Gallery/pull/154
* Misc UI and performance changes for 2.0.0 release (more details soon)

**Full Changelog**: https://github.com/IacobIonut01/Gallery/compare/v1.1.0.1...2.0.0-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/IacobIonut01/Gallery/releases/tag/2.0.0-beta'>Release 2.0.0 Beta</a>.</em>

**Comment by mon-jai** [comment]
Both apks couldn't be installed on Android 14.

![Screenshot_20230916-045337](https://github.com/IacobIonut01/Gallery/assets/91261297/c2ec94b2-ad23-4111-a84a-859a03ce767a)

**Reply by IacobIonut01** [reply]
Re-check your download, just tested and it worked fine (on a14 as well)

**Reply by marieaweirdo** [reply]
Hi, same here, my problem is that it couldn't be installed any of the apks :(( (I'm using Android 10, Sam. A6)

**Reply by IacobIonut01** [reply]
> Hi, same here, my problem is that it couldn't be installed any of the apks :(( (I'm using Android 10, Sam. A6)

Minimum SDK level is API 30 - Android 11

**Reply by marieaweirdo** [reply]
ohhh, that is so sad :(((

**Reply by IacobIonut01** [reply]
> ohhh, that is so sad :(((

This is required because Trash is added in API 30 and most of the new MediaStore apis
